### PR TITLE
Added comments

### PR DESCRIPTION
**Pull request by elken**
Once someone's looked this over, merge.

### CODE REVIEW

**Comment by awslattery** [comment]
image

File: Source/Database.cpp [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 #include <QDebug>
 
+//! Constructor
+/*!
+ * Constructs the local database. 
+ * Currently no interface to handle remote databases, just creates on in the
```
Comment (Hal9007) [inline Source/Database.cpp]: Typo error: on -> one.

Comment (elken) [inline Source/Database.cpp]: Got it

File: Source/Libs/SteamVdfParse.hpp [hunk 2]
```diff
@@ -9,17 +9,28 @@
...
 
+//! Steam VDF Parsing
+/*! A bunch of utility functions to parse the Steam VDF binary format
+*/
 namespace SteamVdfParse
 {
 
 namespace pt = boost::property_tree;
     
+/*! Function to translate an istream object into a 32-bit little endian integer
```
Comment (lolzballs) [inline Source/Libs/SteamVdfParse.hpp]: Does not "translate" an istream, it reads in the integers as big endian and converts them to little endian.

**Pull request merged**